src/app/cheat-sheets/game-base/space-age/space-age.component.html

```diff
@@ -42,10 +42,10 @@ <h4 class="card-title" id="space-platforms">
     </h4>
 
     <ul>
-      <li>Starting planet (Nauvis) has no asteroids to worry about, so orbital platforms do not need defenses</li>
-      <li>You can connect asteroid collectors to circuit network to set filters on what they can grab so you don't overflow on material</li>
+      <li>Starting planet (Nauvis) has no asteroids to worry about, so stationary platforms there do not need defenses.</li>
+      <li>You can connect asteroid collectors to a circuit network to set filters for what they can grab so that your platform doesn't overflow on material.</li>
```

Comment:
```suggestion
      <li>Connect your asteroid collectors to a circuit network, and set limits for each type of asteroid chunk. Now your collectors grab only grab the asteroid chunks you need.</li>
```

How about this? The main goal is to tell users that they can use the circuit network to filter how _many_, and which _type_, of asteroid chunks the collectors should grab.

Reply:
Not every space platform needs to do that, plus it doesn't explain how for non-circuit enthusiasts.